tests/examples/auctions/test_simple_open_auction.py

```diff
@@ -1,77 +1,76 @@
 import pytest
-import ethereum.utils as utils
 
-FIVE_DAYS = 432000
+EXPIRY = 16
 
 
 @pytest.fixture
-def auction_tester(t):
-    from vyper import compiler
-    t.languages['vyper'] = compiler.Compiler()
-    contract_code = open('examples/auctions/simple_open_auction.v.py').read()
-    t.c = t.s.contract(contract_code, language='vyper', args=[t.accounts[0], FIVE_DAYS])
-    return t
+def auction_contract(w3, get_contract):
+    with  open('examples/auctions/simple_open_auction.v.py') as f:
+        contract_code = f.read()
+        contract = get_contract(contract_code, *[w3.eth.accounts[0], EXPIRY])
+    return contract
 
 
-def test_initial_state(auction_tester):
+def test_initial_state(w3, tester, auction_contract):
     # Check beneficiary is correct
-    assert utils.remove_0x_head(auction_tester.c.beneficiary()) == auction_tester.accounts[0].hex()
+    assert auction_contract.beneficiary() == w3.eth.accounts[0]
     # Check bidding time is 5 days
-    assert auction_tester.c.auction_end() == auction_tester.s.head_state.timestamp + 432000
+    assert auction_contract.auction_end() == tester.backend.chain.header.timestamp + EXPIRY - 1
```

Comment:
*if* you want to be future proof, I'd recommend changing this to not reach into the backend and to instead use` tester.get_block_by_number['latest']['timestamp']`.  In fact, now that I've seen this, I'm inclined to rename the location of backend to `tester._backend` so that it's clear that it's a private API.